src/Files/UserControls/MultitaskingControl/MultitaskingControl.xaml.cs

```diff
@@ -0,0 +1,176 @@
+﻿using Files.Backend.Helpers;
+using Files.Backend.ViewModels.Shell.Multitasking;
+using Files.Backend.ViewModels.Shell.Tabs;
+using Microsoft.Toolkit.Uwp;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Linq;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+
+namespace Files.UserControls.MultitaskingControl
+{
+    public sealed partial class MultitaskingControl : UserControl
+    {
+
+        private TabViewItem hoveredTabViewItem = null;
+
+        public MultitaskingControlViewModel ViewModel
+        {
+            get => (MultitaskingControlViewModel)GetValue(ViewModelProperty);
+            set => SetValue(ViewModelProperty, value);
+        }
+
+        // Using a DependencyProperty as the backing store for ViewModel.
+        public static readonly DependencyProperty ViewModelProperty =
+            DependencyProperty.Register(nameof(ViewModel), typeof(MultitaskingControlViewModel), typeof(MultitaskingControl), new PropertyMetadata(null));
+
+        public Visibility TabStripVisibility
+        {
+            get { return (Visibility)GetValue(TabStripVisibilityProperty); }
+            set { SetValue(TabStripVisibilityProperty, value); }
+        }
+
+        // Using a DependencyProperty as the backing store for TabStripVisibility.  This enables animation, styling, binding, etc...
+        public static readonly DependencyProperty TabStripVisibilityProperty =
+            DependencyProperty.Register("TabStripVisibility", typeof(Visibility), typeof(HorizontalMultitaskingControl), new PropertyMetadata(Visibility.Visible));
+
+
+        private DispatcherTimer tabHoverTimer;
```

Comment:
Move to top, _tabHoverTimer;
https://files.community/docs/contributing/code-style